### PR TITLE
Home work20

### PR DESCRIPTION
**Pull request by Angelina0504**
Deleted all the Thread.sleep

### CODE REVIEW

**Comment by Berichicko** [comment]
Please finish the homework!You got this problem, please correct it.

image

**Comment by Angelina0504** [comment]
Homework 19 and 20 both work, both test are passed? I don't understand what is the problem with this two tests

**Comment by Angelina0504** [comment]
Please check again

File: src/test/java/HomeWork20.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class HomeWork20 extends BaseTest{
+
+        @Test
+        public void deletePlaylist() throws InterruptedException{
```
Comment (Berichicko) [inline src/test/java/HomeWork20.java:10]: Please, delete `throws InterruptedException`

**Pull request merged**